openquakeplatform/vulnerability/admin.py

```diff
@@ -0,0 +1,1163 @@
+        'fields': ('typology_repr',
+                   'site_specific',
+                   'hazard_model', )
+    }),
+)
+
+
+qrs_empirical_fieldsets = (
+    (None, {
+        'fields': ('owner',)
+    }),
+    ('Data quality', {
+        'fields': ('data_quantity',
+                   'constrained_categories',
+                   'excitation_observation',
+                   'damage_loss_observation',)
+    }),
+    ('Rationality', {
+        'fields': ('hindcasting',
+                   'cross_validation',
+                   'first_principles',
+                   'uncertainties_treatment',)
+    }),
+    ('Documentation quality', {
+        'fields': ('documentation_quality',
+                   'documentation_type', )
+    }),
+    ('Representativeness', {
+        'fields': ('representativeness', ),
+    }),
+)
+
+
+class CC_PredictorVarInline(OwnedNestedInline, NestedStackedInline):
+    form = CC_PredictorVarForm
+    model = CC_PredictorVar
+    fk_name = 'capacity_curve_func'
+
+
+class CC_AnalyticalModelInfoInline(OwnedNestedInline, NestedStackedInline):
+    model = CC_AnalyticalModelInfo
+    fk_name = 'capacity_curve_func'
+
+
+class CapacityCurveFuncInline(OwnedNestedInline, NestedStackedInline):
+    model = CapacityCurveFunc
+    inlines = [
+        CC_AnalyticalModelInfoInline,
+        CC_PredictorVarInline,
+    ]
+
+
+class DL_FuncDistrDTLDiscrInline(OwnedNestedInline, NestedStackedInline):
+    model = FuncDistrDTLDiscr
+    fl_name = 'damage_to_loss_func'
+
+
+class DamageToLossFuncInline(OwnedNestedInline, NestedStackedInline):
+    model = DamageToLossFunc
+    inlines = [
+        DL_FuncDistrDTLDiscrInline,
+    ]
+
+
+class VU_AnalyticalModelInfoInline(OwnedNestedInline, NestedStackedInline):
+    model = AnalyticalModelInfo
+    exclude = ('fragility_func',)
+    fk_name = 'vulnerability_func'
+
+
+class VU_EmpiricalModelInfoInline(OwnedNestedInline, NestedStackedInline):
+    model = EmpiricalModelInfo
+    exclude = ('capacity_curve_func',
+               'damage_to_loss_func',
+               'fragility_func')
+    fk_name = 'vulnerability_func'
+
+
+class VU_StatisticalInformationInline(OwnedNestedInline, NestedStackedInline):
+    model = StatisticalInformation
+    exclude = ('fragility_func', )
+    fk_name = 'vulnerability_func'
+
+
+class VU_PredictorVarInline(OwnedNestedInline, NestedStackedInline):
+    model = PredictorVar
+    form = PredictorVarForm
+    exclude = ('fragility_func',)
+    fk_name = 'vulnerability_func'
+
+
+class VU_FuncDistrVulnDiscrInline(OwnedNestedInline, NestedStackedInline):
+    model = FuncDistrVulnDiscr
+    fk_name = 'vulnerability_func'
+
+
+class VU_FuncDistrVulnContInline(OwnedNestedInline, NestedStackedInline):
+    model = FuncDistrVulnCont
+    fk_name = 'vulnerability_func'
+
+
+def formset_by_rel_name(formsets, rel_name):
+    for formset in formsets:
+        #print("prima")
+        #print(formset.fk.related.get_accessor_name())
+        #print("dopo")
```

Comment:
Please remove the three lines above.

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/19/commits/f440b714cebf6d76ac9232655e33cbfe5237d8a0